tests/integration/initramfs.bats

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function setup() {
+	INITRAMFS_ROOT="$(mktemp -d "$BATS_RUN_TMPDIR/runc-initramfs.XXXXXX")"
+}
+
+function teardown() {
+	[ -v INITRAMFS_ROOT ] && rm -rf "$INITRAMFS_ROOT"
+}
+
+function qemu_native() {
+	# Different distributions put qemu-kvm in different locations and with
+	# different names. Debian and Ubuntu have a "kvm" binary, while AlmaLinux
+	# has /usr/libexec/qemu-kvm.
+	qemu_candidates=( "kvm" "qemu-kvm" "/usr/libexec/qemu-kvm" )
+
+	qemu_binary=
+	for candidate in "${qemu_candidates[@]}"; do
+		"$candidate" -help &>/dev/null || continue
+		qemu_binary="$candidate"
+		break
+	done
+	# TODO: Maybe we should also try to call qemu-system-FOO for the current
+	# architecture if qemu-kvm is missing?
+	[ -n "$qemu_binary" ] || fail "could not find qemu binary"
+
+	sane_run "$qemu_binary" "$@"
+}
+
+@test "runc run [initramfs + pivot_root]" {
+	requires root
```

Comment:
The test should be skipped when QEMU is missing?

Reply:
Also check that /boot/vmlinuz is present and the host arch is `x86_64`

Reply:
I've adjusted the tests, it should work on arm64 as well now (running CI to double-check now). I've also made the test get skipped if we can't find qemu.

Reply:
Ah, [Actuated don't support aarch64 guests](https://docs.actuated.com/examples/kvm-guest/) (and I'm not even sure if we have nested virtualisation support enabled, not to mention that the kernel is missing from `/boot`). I'll just skip this test for actuated, but I'll keep the code as arch-agnostic as possible.